helm/fluent-logshipping-app/templates/configmap.yaml

```diff
@@ -67,27 +74,27 @@ data:
   inputs-syslog.conf: |
     [INPUT]
         Name              systemd
-        Path              /run/log/journal
         Tag               syslog.*
-        Parser            syslog
         DB                ${input_database_path}/syslog.db
         storage.type      {{ .Values.fluentbit.inputStorageTypes.syslog }}
         Mem_Buf_Limit     ${input_memory_buffer_limit}
         Strip_Underscores on
 
   inputs-auditing.conf: |
+    {{- if has "sshd" .Values.outputs.inputLogTypes }}
     # Get the sshd audit logs
     [INPUT]
         Name              systemd
         Tag               audit.ssh.*
-        Path              /run/log/journal
-        Parser            syslog
         DB                ${input_database_path}/audit_ssh.db
-        Systemd_Filter    SYSLOG_IDENTIFIER=sshd
-        Mem_Buf_Limit     ${input_memory_buffer_limit}
         storage.type      {{ .Values.fluentbit.inputStorageTypes.sshd }}
+        Mem_Buf_Limit     ${input_memory_buffer_limit}
         Strip_Underscores on
+        Systemd_Filter    SYSLOG_IDENTIFIER=sshd
```

Comment:
Supporting both syslog_identifier and systemd_unit. This is an OR relationship by default

Reply:
Maybe add this as a comment in the file.